ros2_batch_job/__main__.py

```diff
@@ -48,7 +48,8 @@
     'pep8',
     'pydocstyle',
     'pyflakes',
-    'flake8',
+    'flake8==2.6.2',
```

Comment:
it's not straightforward to use the new release in https://github.com/ament/ament_lint/pull/63 (there is no 3.x API, and the shim to the 2.x API doesn't expose what's needed).

I appreciate pinning to an old version isn't _ideal_ (especially since one of the selling points of flake8 is its active development). if using 2.x is a _blocker_, I'll put https://github.com/ament/ament_lint/pull/63 on the backburner until there is an API in 3.x since it might take me a bit to get 3.x to work